app/ui-react/packages/ui/src/Integration/Activity/IntegrationDetailActivityItem.tsx

```diff
@@ -1,152 +1,177 @@
-      <>
-        <OkIcon /> Success
-      </>
-    ) : (
-      <>
-        <ErrorCircleOIcon /> Error
-      </>
-    )}
-  </Table.Cell>
-);
-const outputCellFormat = (output: string) => (
-  <Table.Cell className="integration-detail-activity-item__output">
-    <pre className="integration-detail-activity-item__output-step-data">
-      {output || 'No output'}
-    </pre>
-  </Table.Cell>
-);
-
-export class IntegrationDetailActivityItem extends React.Component<
+export const IntegrationDetailActivityItem: React.FC<
   IIntegrationDetailActivityItemProps
-> {
-  public render() {
-    return (
-      <ListView.Item
-        className="integration-detail-activity-item"
-        key={this.props.time}
-        heading={this.props.date}
-        description={this.props.time}
-        additionalInfo={[
-          <ListView.InfoItem key={1}>
-            {this.props.i18nVersion}
-            &nbsp;
-            {this.props.version}
-          </ListView.InfoItem>,
-          <ListView.InfoItem key={2}>
-            <div className="integration-detail-activity-item__status-item">
-              {this.props.errorCount > 0 ? (
+> = (
+  {
+    date,
+    errorCount,
+    i18nError,
+    i18nErrorsFound,
+    i18nHeaderStep,
+    i18nHeaderTime,
+    i18nHeaderDuration,
+    i18nHeaderStatus ,
+    i18nHeaderOutput,
+    i18nNoErrors,
+    i18nNoOutput,
+    i18nNoSteps,
+    i18nSuccess,
+    i18nVersion,
+    steps,
+    time,
+    version
+  } ) => {
+  const [rowExpanded, setRowExpanded] = useState(false);
+
+  const doExpand = () => {
+    setRowExpanded(!rowExpanded);
+  };
+
+  const rows = () => {
+    const newRows = steps.map((step: any) => {
+      return [{
+        cells: [
+          step.name,
+          step.time,
+          step.duration,
+          {
+            title: (
+              step.status === 'Success' ? (
                 <>
-                  <ErrorCircleOIcon />
-                  {'  '}
-                  {this.props.i18nErrorsFound}
+                  <OkIcon size={'sm'} color={global_success_color_100.value} />{'  '}{i18nSuccess}
                 </>
               ) : (
                 <>
-                  <OkIcon />
-                  {'  '}
-                  {this.props.i18nNoErrors}
+                  <ErrorCircleOIcon size={'sm'} color={global_danger_color_100.value} />{'  '}{i18nError}
                 </>
-              )}
-            </div>
-          </ListView.InfoItem>,
-        ]}
+              )
+            )
+          },
+          {
+            title: (
+              <pre className={'integration-detail-activity-item__output-step-data'}>
+                {step.output || i18nNoOutput}
+              </pre>
+            )
+          }
+        ],
+      }];
+    });
+
+    return newRows.reduce((a: any, b: any) => a.concat(b), []);
+  };
+
+  const columns = [
+    i18nHeaderStep,
+    i18nHeaderTime,
+    i18nHeaderDuration,
+    i18nHeaderStatus,
+    i18nHeaderOutput
```

Comment:
I think this is where you'd add those modifiers to increase the width of a few of those columns, i.e.

`const columns = [ { title: i18nHeaderStep, transform: cellWidth(12) }, etc etc ];`

Reply:
btw I remember now that I tried this last week and it seems to have no effect on the cell width, not sure why. Would you be able to test this locally? I've tried variations of the following:

```
const columns = [
    { title: i18nHeaderStep, transform: cellWidth(10) },
    { title: i18nHeaderTime, transform: cellWidth(30) },
    i18nHeaderDuration,
    { title: i18nHeaderStatus, transform: cellWidth('max') },
    i18nHeaderOutput
  ];
```